extensions/ql-vscode/src/helpers.ts

```diff
@@ -775,7 +786,16 @@ export async function askForLanguage(
         "Language not found. Language must be specified manually.",
       );
     }
+    return undefined;
   }
+
+  if (!isQueryLanguage(language)) {
+    void showAndLogErrorMessage(
+      `Language '${language}' is not supported. Language must be specified manually.`,
```

Comment:
Should we be showing the list of supported languages instead of saying `Language must be specified manually`?

Reply:
Does this PR make the extension's validation stricter for local queries, and only allow query runs on languages that are listed in the extension code?

Reply:
It makes validation stricter in some cases, but I believe normal local queries for unsupported languages should still work. What won't work is:
- Creating a skeleton query pack
- Running a MRVA query
- Running a query on multiple databases

Reply:
That sounds fine. We may want to weaken it in future as we expand to more languages, but for now I think that's an acceptable set.